resources/baseJenkins.yaml

```diff
@@ -100,7 +87,7 @@ unclassified:
         showMatrixStatus: false
     githubAuth:
     - description: "Anonymous connection"
-      id: "fe7046c5-29a0-4370-b32d-47aead7c5945"
+      id: "d4456c70-9c5e-4b40-bee4-e1ebb693153b"
```

Comment:
Is this value constant?

Reply:
I think its random, not affecting though.

Reply:
I think we should then add a comment here for it, since it seems like this is not the first time it would be changing.